### PR TITLE
📖Callout for amp-video-iframe samples

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: extensions/amp-video-iframe/amp-video-iframe.md [hunk 1]
```diff
@@ -191,6 +191,8 @@ The `src` and `poster` URLs are appended with [`data-param-*` attributes as quer
 
 The `/amp-video-iframe` document bootstraps the [integration script](#integration) so that the AMP document can coordinate with the player.
 
+Note: If you're hosting an integration document for users of your video service, feel free to [contribute a code sample to this page.](https://github.com/ampproject/amphtml/blob/master/extensions/amp-video-iframe/amp-video-iframe.md)
+
 Note: For most video providers, `amp-video-iframe` provides enough tools for common playback actions (see [methods](#method) and [events](#postEvent)). Refer to the [vendor player spec](../../spec/amp-3p-video.md) for more details on whether you can use `amp-video-iframe` or you should build a third-party player component instead.
```
Comment (CrystalOnScript) [inline extensions/amp-video-iframe/amp-video-iframe.md]: We probably shouldn't have two notes on top of each other. Lol. I would suggest ordering this section by what you want readers to know first. I think that we want them to host an integration of their service. Maybe we should start the `Third-party video vendors` section with something like:

`AMP accepts vendor integrations in the form of configuration.....`

I'm struggling with the proper wording. But if you get my drift, WDYT?

File: spec/amp-3p-video.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ The generic [`amp-video`](https://go.amp.dev/c/amp-video) component plays videos
 Unfortunately, not all videos can be embedded this way. For these specialized cases, AMP provides vendor-optimized components like [`amp-youtube`](https://go.amp.dev/c/amp-youtube), [`amp-ima-video`](https://go.amp.dev/c/amp-ima-video), and [others](./amp-video-interface.md).
 Internally these players load an iframe whose page communicates with the outer document to coordinate playback.
 
-We would prefer to not have additional custom video implementations. Instead we believe, [`amp-video-iframe`](https://go.amp.dev/c/amp-video-iframe) should be used instead, since it is a generic component that can load any video document to coordinate playback.
+**We would prefer to not have additional custom video implementations.** Instead we believe, [`amp-video-iframe`](https://go.amp.dev/c/amp-video-iframe) should be used instead, since it is a generic component that can load any video document to coordinate playback.
```
Comment (CrystalOnScript) [inline spec/amp-3p-video.md]: Same as previous comment about ordering documentation. Can we also change the wording to something more opinionated? Like:
`We are no longer accepting custom video implementations in the form of AMP components. Instead, new implementations should com int he form of integrations with amp-video-iframe. If you believe you are unable to integrate with amp-video-iframe something something contact here?`

File: extensions/amp-video-iframe/amp-video-iframe.md (hunk 1 above)
Comment (alanorozco) [reply extensions/amp-video-iframe/amp-video-iframe.md]: Restructured, let me know if this flows better.

File: spec/amp-3p-video.md (hunk 2 above)
Comment (alanorozco) [reply spec/amp-3p-video.md]: Added blurb about contact.

How do you see this being better restructured? Unsure about what the sub-sections are here.

About the wording with `we are no longer accepting custom video implementations in the form of AMP components`, that's stronger than the current policy (we're considering #26572 as an alternative, for example)

**Pull request merged**